app/views/admin/loans/content/_questionnaire_group.html.slim

```diff
@@ -98,31 +98,52 @@
-                  .view-element.doc-actions
-                    - if document.display_url.present?
-                      / Beware, the display_height value doesn't not seem to be honored
-                      a[href="#" data-expands="iframe-#{field.attribute_sym}"
-                        data-content="<iframe src='#{document.display_url}' height='#{document.display_height}'>"]
-                        i.fa.fa-eye.fa-large>
-                        = t("loan.spreadsheet_inline")
-                      a href="#" data-hides="iframe-#{field.attribute_sym}"
-                        i.fa.fa-eye-slash.fa-large>
-                        = t("loan.spreadsheet_hide")
-                      = link_to document.url, target: '_blank'
-                        i.fa.fa-external-link.fa-large>
-                        = t("loan.spreadsheet_new_tab")
-                div.spreadsheet data-expandable="iframe-#{field.attribute_sym}"
+              .attached-document
+                .linked-document.form-element.form-inline
+                  h5 = t("linked_document.linked_document")
+
+                  .ld-container
+                    .form-group.full-width-group
+                      label for="#{field.attribute_sym}__url" = t("linked_document.url")
+                      = f.input_field "#{field.attribute_sym}__url".to_sym, as: :string,
+                        class: 'form-control', placeholder: t("linked_document.link")
+                    .help-block
+                      span = t("linked_document.range_of_cells")
+                    .form-group
+                      label for="#{field.attribute_sym}__start_cell" = t("linked_document.cell_start")
+                      = f.input_field "#{field.attribute_sym}__start_cell".to_sym, as: :string,
+                      class: 'form-control', placeholder: t("linked_document.cell_start_hint")
+                    .form-group
+                      label for="#{field.attribute_sym}__end_cell" = t("linked_document.cell_end")
+                      = f.input_field "#{field.attribute_sym}__end_cell".to_sym, as: :string,
+                      class: 'form-control', placeholder: t("linked_document.cell_end_hint")
+                    - if document
+                      - if document.display_url.present?
+                        .doc-actions
+                          a [data-action="delete"
+                            data-confirm== t("linked_document.delete_confirmation")
+                            data-confirm-cancel== t(:cancel)
+                            data-confirm-proceed== t(:confirm)
+                            data-confirm-title== t("linked_document.linked_document")]
+                            = t("linked_document.delete")
+
+                .spreadsheet-links.view-element
+                  - if document
+                    .doc-actions
+                      - if document.display_url.present?
+                        / Beware, the display_height value doesn't not seem to be honored
```

Comment:
It is. That was Joseph's note. And he had the display_height commented out and the box set to a height of about 500px.

The display_height only changes for content if both a starting cell and ending cell are defined. If not, the height is big. Even with those start and end cells, the height still isn't exactly the height of the content, but is a tad bit bigger. This only applies to spreadsheets (being able to adjust the height with start and end cell).